exercises/julia-1-a/README.md

```diff
@@ -0,0 +1,98 @@
+## Instructions
+
+Implement your own [tic-tac-toe](https://en.wikipedia.org/wiki/Tic-tac-toe) game.
```

Comment:
I think the task is to implement a function to determine "state" of game, not create the game itself (which to me implied flow and state, etc).

Other than that, this is good! :)

Reply:
```suggestion
In this exercise, you will implement a computer-controlled [tic-tac-toe](https://en.wikipedia.org/wiki/Tic-tac-toe) opponent.
In the first part, you will write a program that analyses the state of the game.
```

Is this better?

Reply:
No :) 

We're not telling people there **is** a second part, because that then makes them code with future requirements in mind, which invalidates the research. So all we want to mention is the state of game bit :)

Reply:
Ah, I didn't know that. How about this?

```suggestion
In this exercise, you're going to implement a program that determines the state of a [tic-tac-toe](https://en.wikipedia.org/wiki/Tic-tac-toe) game.
```

Reply:
Thanks :)